common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -218,13 +218,9 @@ def student_view(self, context):
             rendered_child = child.render(STUDENT_VIEW, context)
             fragment.add_frag_resources(rendered_child)
 
-            # `titles` is a list of titles to inject into the sequential tooltip display.
-            # We omit any blank titles to avoid blank lines in the tooltip display.
-            titles = [title.strip() for title in child.get_content_titles() if title.strip()]
             childinfo = {
                 'content': rendered_child.content,
-                'title': "\n".join(titles),
-                'page_title': titles[0] if titles else '',
+                'page_title': child.display_name_with_default if child.display_name_with_default else '',
```

Comment:
How concerned are we about the titles changing when this is deployed? From looking into test failures it seems that page_title is now different for at least some of the test courses, which indicates that it could change what is rendered on the actual site (ie the new title in the tooltip might not be the same as the first title in the old list of titles displayed in the tooltip). @nasthagiri